Form/Type/SeoMetadataType.php

```diff
@@ -69,8 +76,15 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
      */
     public function configureOptions(OptionsResolver $resolver)
     {
+        $class = null;
+        if ($this->container->has('cmf_seo.backend_type_phpcr')) {
```

Comment:
if somebody is using doctrine orm or something else, they might want to
overwrite it. but we can start without making it configurable, it can
still be overwritten when using the form type.